tools/ci_build/github/android/build_aar_and_copy_artifacts.sh

```diff
@@ -8,6 +8,15 @@ set -e
 set -x
 export PATH=/opt/python/cp38-cp38/bin:$PATH
 
+EXTRA_ARG=""
+while getopts "x:" parameter_Option
```

Comment:
"x:" means the parameter must take a value, which means you must always have something in EXTRA_ARG.  If you don't, it will eat the next parameter as the value of "-x". 

As we do not need to provide two different packages: with azureep and without azureep, you may just hardcode this thing at somewhere. You do not need to pass it in. 

If you really need to pass it in, please fix the issue above.